uaclient/security_status.py

```diff
@@ -26,20 +30,23 @@ class UpdateStatus(Enum):
     UNAVAILABLE = "upgrade_unavailable"
 
 
-def get_service_name(origins: List[apt_package.Origin]) -> str:
+def is_esm_package(package: apt_package.Package, service_name: str):
+    return any(
+        (origin.origin, origin.archive)
+        == SERVICE_TO_ORIGIN_INFORMATION[service_name]
+        for origin in package.installed.origins
+    )
+
+
+def get_service_name(origins: List[apt_package.Origin]) -> Tuple[str, str]:
     "Translates the archive name in the version origin to a UA service name."
     for origin in origins:
-        if (
-            "infra-security" in origin.archive
-            and origin.origin == ESM_INFRA_ORIGIN
-        ):
-            return "esm-infra"
-        if (
-            "apps-security" in origin.archive
-            and origin.origin == ESM_APPS_ORIGIN
-        ):
-            return "esm-apps"
-    return "standard-security"
+        service = ORIGIN_INFORMATION_TO_SERVICE.get(
```

Comment:
Should we have some preference here ?

I am assuming the following scenario: We have a package that for some reason has the same version delivered on `esm-infra` and `esm-apps`. In that situation, we would choose randomly which package to origin comes first

Reply:
Indeed. This is a good question. Although I don't foresee any problem showing any of them, this seems like it's worth deeper thought on what we want to prioritize.

Reply:
I think this is unlikely because esm-apps is for universe packages and esm-infra is for main packages, right?

Reply:
It should be, yes, but I think we do have same packages that collide. I think `krb5-admin-server` is an example that appear on both esm security pockets.

However, maybe this is just a corner case that we don't need to properly address

Reply:
As discussed, we can assume versions are always different across official origins so we don't have to worry here